### ISSUE TITLE
Low light enhancement

### ISSUE THREAD

**Issue by Euanggelos** [chaiNNer-org/spandrel#288]
Hi
Could you include support for  some of the models included in this link?
https://github.com/caiyuanhao1998/MST-plus-plus
specially for higher PSNR? Restormer MPRNet MST-L MST++

**Comment by RunDevelopment** [comment]
We should already support Restormer and MPRNet.

About MST and MST++: These are meant to reconstruct for full spectrum from an RGB image, AFAIK. So given an RGB image, they will output an approximation of the light frequency spectrum for each pixel. I guess this could be used for Low-light enhancement, but not directly as is. 

So I don't know which specific models you want us to support. @Euanggelos, could you please link them more directly?

**Comment by Disonantemus** [comment]
> We should already support Restormer and MPRNet.

I tried both today from [spandrel Readme](https://github.com/chaiNNer-org/spandrel) links, using ChaiNNer:  

- **Restormer**: No Low-light model; I did test 4 (3 worked, from 13), but did other things like: deblurring, denoising. I did find [restormer.pth](https://drive.google.com/file/d/1tDwe9X46bfaRrdTnQi4q2ZPnwssA5B2A/view), did load, but finish with an error.
- **MPRNet**: No Low-light model, I did test 2 models (from 3) and worked, as expected doing deblurring and denoise. I did find [mprnet.pth](https://drive.google.com/file/d/1amYEpxlBnT1pmk7JmKWYT_bi7WAJH-ad/view), but doesn't load as a model.

Maybe both doesn't have Low-Light Enhancement models, and are in the [Results on NTIRE 2022 HSI Dataset](https://github.com/caiyuanhao1998/MST-plus-plus) just to compare older models.

---

From [NTIRE 2024 Challenge on Low Light Enhancement](https://codalab.lisn.upsaclay.fr/competitions/17640), seems that **RetinexFormer** and **MST++** are nice aditions, because they ranked top-2.

From Readme, RetinexFormer **is supported**, but models from links says unsupported in ChaiNNer. Maybe ChaiNNer (0.24.1) doesn't have an updated spandrel that support this.

---

In summary, I **didn't find any Low-Light models in ChaiNNer** that works.

---

~~~ocaml
chainner-bin (Arch AUR): 0.24.1-1 (Last Release)
distro: Arch Linux x86_64
kernel: 6.6.22-1-lts
shell: bash 5.2.26
term: tmux
cpu: Intel i7-4790 (8) @ 3.600GHz
gpu: AMD ATI Radeon RX 470/480/570/570X/580/580X/590
~~~

**Comment by Disonantemus** [comment]
- Maybe we need to wait for next update (Spandrel and/or ChaiNNer)?
- Is possible to compile Spandrel to get/test newer commits? (I don't know how, not a programmer)
